packages/wp-now/README.md

```diff
@@ -89,211 +91,234 @@ cd wordpress-plugin-or-theme
+You can find the `example.log` file in your project's directory.
 
-### wp-env
+## Known issues
+
+-   Running `wp-now start` in 'wp-content' or 'wordpress' mode creates empty directories. See [GitHub issue #32](https://github.com/WordPress/playground-tools/issues/32).
+-   The `wp-now` NPM package may appear to have a random version number. See [GitHub issue #31](https://github.com/WordPress/playground-tools/issues/31).
 
-If you are migrating from `wp-env`, you should be aware of the differences it has with `wp-now`:
+## Comparisons
 
--   `wp-now` supports non-WordPress projects;
--   `wp-now` does not need Docker;
--   `wp-now` does not support Xdebug PHP extension (yet!);
--   `wp-now` does not include Jest for automatic browser testing
+### Laravel Valet
+
+Here's what you need to know if you're migrating from Laravel Valet:
+
+-   `wp-now` handles the entire WordPress installation for you. Just run the `wp-now start` command.
+-   `wp-now` works across all desktop platforms (Mac, Linux, Windows).
+-   `wp-now` does not set up custom domains for you.
+-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed (see item #1 above).
+-   `wp-now` allows you to switch the WordPress version with `wp-now start --wp=VERSION_NUMBER`(replace `VERSION_NUMBER` with the actual WordPress version).
+
+### wp-env
 
-Some similarities between `wp-env` and `wp-now` to be aware of:
+Here's what you need to know if you're migrating from `wp-env`:
 
--   no support for custom domains or SSL;
--   `plugin`, `themes` and index modes are available on `wp-env` and `wp-now`;
--   deployments are not possible with neither `wp-env`, nor `wp-now`;
--   possible to switch easily the PHP version
+-   `wp-now` supports non-WordPress projects.
+-   `wp-now` does not require Docker.
+-   `wp-now` does not include Jest for automatic browser testing.
 
 ## Contributing
 
 We welcome contributions from the community!
 
-In order to contribute to `wp-now`, you'll need to first install a few global dependencies:
+### Code contribution
 
--   Make sure you have `nvm` installed. If you need to install it first,
-    [follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
--   Install `nx` by running `npm install -g nx`.
-
-Once the global dependencies are installed, you can start using the repo:
+Clone the repository, install the dependencies, build the project, and start `wp-now` in `preview` mode:
 
 ```bash
 git clone git@github.com:WordPress/playground-tools.git
 cd playground-tools
-nvm use
 npm install
 npm run build
-nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
+npx nx preview wp-now start
 ```
 
-If you'd like to make the `wp-now` executable globally available when using this installation method, run `npm link`. It's not particularly reliable, however.
-
-## Testing
-
 To run the unit tests, use the following command:
 
 ```bash
-nx test wp-now
-```
-
-## Publishing
```

Comment:
If this gets removed, the next person trying to publish the package won't have the documentation. It would be lovely to either preserve it or move it to another file.

Reply:
Oops, I assumed that this is an internal process that's also covered somewhere else and accidentally found its way to the public page :-)

--@adamziel How about creating a separate `PUBLISHING.md` file that includes this section and other information relevant for admins?

Reply:
Yeah!, moving it to a different file sounds good to me! 👌

Reply:
Coolio :-) Done: [6a5f7b0](https://github.com/WordPress/playground-tools/pull/251/commits/6a5f7b0aac7102d69ac483131ff68f4c0d9a572d)